lib/plurimath/math/function/underover.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Plurimath
+  module Math
+    module Function
+      class Underover
```

Comment:
The code for `Underover` and `PowerBase` is identical. Can we DRY this with either a common superclass or using one as a superclass/subclass?